window.go

```diff
@@ -116,9 +117,19 @@ type WindowOptions struct {
 	Y                      *int            `json:"y,omitempty"`
 
 	// Additional options
-	Custom *WindowCustomOptions `json:"custom,omitempty"`
-	Load   *WindowLoadOptions   `json:"load,omitempty"`
-	Proxy  *WindowProxyOptions  `json:"proxy,omitempty"`
+	Custom     *WindowCustomOptions `json:"custom,omitempty"`
+	Load       *WindowLoadOptions   `json:"load,omitempty"`
+	Proxy      *WindowProxyOptions  `json:"proxy,omitempty"`
+	AppDetails *WindowAppDetails    `json:"appDetails,omitempty"`
```

Comment:
Could you move `AppDetails` above `Custom` so that additional options are sorted alphabetically?